src/main/resources/representative-details/translation.json

```diff
@@ -20,26 +22,27 @@
       "fullNameLabel": "Enw llawn",
       "organisationNameLabel": "Enw’r sefydliad neu’r busnes",
       "contactNumberLabel": "Rhif cyswllt",
-      "emailAddressLabel": "Cyfeiriad e-bost"
+      "contactNumberHint": "Enter a telephone number in the correct format, like 01632 960 001 or 07700 900 982 (WELSH)",
```

Comment:
Why not using the google translator to convert to welsh?

Reply:
Because Welsh translation is out of the scope of this ticket (as declared in the description of this PR), and I do not have a sufficient enough knowledge of Welsh myself to be able to trust that Google Translate provides an accurate enough translation